plugins/net_plugin/include/eosio/net_plugin/net_plugin.hpp

```diff
@@ -13,6 +13,24 @@ namespace eosio {
       handshake_message last_handshake;
    };
 
+   using namespace plugin_interface;
+
+   struct net_plugin_metrics : plugin_metrics {
+      runtime_metric num_peers{metric_type::gauge, "num_peers", "num_peers", 0};
```

Comment:
Is it useful to have `label` and `family` the same? What if you just leave one as `""` does that cause a problem?